audio/out/ao_openal.c

```diff
@@ -199,8 +199,12 @@ static int init(struct ao *ao)
     alListenerfv(AL_ORIENTATION, direction);
 
     alGenSources(1, &source);
-    if (p->direct_channels && alGetEnumValue((ALchar*)"AL_DIRECT_CHANNELS_SOFT")) {
-        alSourcei(source, alGetEnumValue((ALchar*)"AL_DIRECT_CHANNELS_SOFT"), AL_TRUE);
+    if (p->direct_channels) {
+        if (alIsExtensionPresent("AL_SOFT_direct_channels_remix")) {
+            alSourcei(source, alGetEnumValue((ALchar*)"AL_DIRECT_CHANNELS_SOFT"), 2);
```

Comment:
What does `2` mean here as opposed to `AL_TRUE`?

Reply:
2 is equal to [AL_REMIX_UNMATCHED_SOFT](https://openal-soft.org/openal-extensions/SOFT_direct_channels_remix.txt).